server/controller/chore.js

```diff
@@ -261,11 +261,11 @@ export const checkOffChore = async (req, res) => {
         .json({ message: `Chore with id ${id} was not found` });
     }
 
-    checkOffTime = Date.now();
+    const checkOffTime = Date.now();
     // adding the check off time in the array of chores last checked off list
     chore.lastCheckedOff.push(checkOffTime);
 
-    const repeatMs = repeatInMs(chore.quantity, chore.interval);
+    const repeatMs = repeatInMs(chore.frequency.quantity, chore.frequency.interval);
```

Comment:
I thought I changed this in my PR? Did you pull develop? It looks like half my PR made its way into your codebase somehow haha

Reply:
Yup I did pull develop. This is weird but whatever lol
It works now

Reply:
Yeah, I forgot to push my server-side stuff. Could you revert the changes you made to the front-end and modify the backend routes to reflect that? Or I could make a new PR to get the stuff I changed in.